Core/TraceLogger.h

```diff
@@ -52,7 +52,7 @@ struct TraceLoggerOptions
 class TraceLogger
 {
 private:
-	static constexpr int ExecutionLogSize = 30000;
+	static constexpr int ExecutionLogSize = 20000;
```

Comment:
What are we changing ExecutionLogSize for?

Reply:
The compiler is giving this error after a VS9 update:
fatal error C1060: compiler is out of heap space (compiling source file TraceLogger.cpp)
I find out the error is caused by this value. The value is used to create several large arrays which are not dynamically allocated and the compiler is holding up heap space for them when building. I'm not sure how to increase heap space in VS9 and this is the only solution I can find so far.

Reply:
That makes sense, and that's fine with me